src/app/components/core/developer/additional-services/additional-services-utils.service.ts

```diff
@@ -83,7 +83,7 @@ export class AdditionalServicesUtils {
         (data: any) => {
           let servicesRegistry = data.services.filter((el) => expectedExternalServiceType.includes(el.type))
           if (from !== 'additional-services') {
-            servicesRegistry = data.services.filter((s => (s.type.toLowerCase() === from)));
+            servicesRegistry = data.services.filter((s => (s.type.toLowerCase() === (from === 'bucket' ? 'bucketstorage' : from))));
```

Comment:
bucket is not additional-service? the condition above says `if (from !== 'additional-services') `

Reply:
Bucket is additional service but here we are checking from where this function is being called i.e. from `additional-services` page or any other service page like- `notification`, `bucket` etc.